packages/modules/cookies/js/content.js

```diff
@@ -0,0 +1,24 @@
+/* global chrome, locationFilterMatches */
+
+(function () {
+  function recordCookies() {
+     const url = window.location.href;
+    // if (locationFilterMatches(window.location, config['cookie-filters'])) {
+      if (url === 'about:blank' || !url || url === '') {
```

Comment:
|| url === '' it's useless check because you already have !url.
Please change the order of the checks: if (!url || url === 'about:blank')